.github/workflows/ci.yml

```diff
@@ -14,10 +14,13 @@ jobs:
     - uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.nodejs }}
+        
+    - name: Update npm
+      run: npm install -g npm@latest
```

Comment:
why are you updating npm here? shouldn't the npm version be configured via the `setup-node` action?

it looks like this job uses the matrix strategy to test multiple node versions, and doesn't cache `npm`, so presumably we want to test against each version of `npm` shipped with the corresponding node version we test against?

Reply:
The npm version packed with node 12 and 14 is too low to support lockfile v2